docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -604,7 +604,7 @@ is assumed that different medications have a similar impact and therefore are no
 
 
 Decrease in SBP is dependent on a simulant's starting SBP value. Full efficacy data is here:
-/share/scratch/projects/cvd_gbd/cvd_re/simulation_science/drug_efficacy_sbp_new.csv [Law_2009]_
+/mnt/share/homes/sbachmei/repos/vivarium_nih_us_cvd/src/vivarium_nih_us_cvd/data/drug_efficacy_sbp.csv [Law_2009]_
```

Comment:
Nicer to link to this through GitHub, I think: https://github.com/ihmeuw/vivarium_nih_us_cvd/blob/main/src/vivarium_nih_us_cvd/data/drug_efficacy_sbp.csv

Reply:
Yes, @zmbc is correct that way people aren't relying on me and/or my folder permissions to access this. If in the future this dataset needs to change, it can be done so through the repo itself

Reply:
This is nicer!! Thanks :)